_data/hosts.yml

```diff
@@ -858,6 +858,37 @@
             version: '5.5.??'
             semver: '5.5.??'
 -
+    name: Liquidweb
+    url: 'http://www.liquidweb.com/?RID=phpversions'
+    type: managed
+    default: 55
+    versions:
+        53:
+            phpinfo: null
```

Comment:
I'm trying to avoid adding more hosts with unknown patch numbers of missing phpinfos, it's adding more noise than value I think. Could you hunt these down?